Logicless-Coder/README.md

```diff
@@ -0,0 +1,13 @@
+## GOALS:-
+<b>Competitive Programming -</b>
+- [ ] Reach a rank of 4 kyu and atleast 1000 Honor on Codewars
```

Comment:
Hey Abhishek!
I would prefer if you write the whole readme in `markdown`, even though html is accepted.
Eg. for **bold**, use two asterix(*)